overzoom.cpp

```diff
@@ -158,6 +163,25 @@ int main(int argc, char **argv) {
 			do_compress = false;
 			break;
 
+		case 'k' & 0x1F: {
+			clipbbox clip;
+			if (sscanf(optarg, "%lf,%lf,%lf,%lf", &clip.lon1, &clip.lat1, &clip.lon2, &clip.lat2) == 4) {
+				projection->project(clip.lon1, clip.lat1, 32, &clip.minx, &clip.maxy);
+				projection->project(clip.lon2, clip.lat2, 32, &clip.maxx, &clip.miny);
+				clipbboxes.push_back(clip);
+			} else {
+				fprintf(stderr, "%s: Can't parse bounding box --clip-bounding-box=%s\n", argv[0], optarg);
+				exit(EXIT_ARGS);
+			}
+			break;
+		}
+
+		case 'l' & 0x1F: {
+			clipbbox clip = parse_clip_poly(optarg);
```

Comment:
Clip the clip regions to the tile bounds to speed up the case where complicate bounds entirely enclose the tile.